REQUIRE

```diff
@@ -1,2 +1,3 @@
 julia 0.6
-IterativeSolvers 0.4.1
+IterativeSolvers 0.7.0
```

Comment:
We should probably use v0.6.0 because IterativeSolvers 0.7.0 needs Julia 0.7

Reply:
Actually more stuff need to be changed, I was running the tests wrongly. This too yes.

Reply:
Cool, ping me when you're done making all those. We should add v0.7 to `.travis.yml` as well. Thanks a lot for this PR! There was a lot of dead code on the classical AMG which I didn't have time to get to.